docs/readthedocs/source/doc/UserGuide/contributor.rst

```diff
@@ -0,0 +1,5 @@
+Contributor Guide
+=========================
+
+* `Developer Guide <./develop.html>`_ provides instructions to edit BigDL source code
+* `Documentation Guide <./documentation.html>`_ lists multiple tips and rules for revising or adding documents to BigDL documentation. 
```

Comment:
Documentation Guide provides tips and guidelines for adding/modifying BigDL documents.